pkg/user/service/userservice.go

```diff
@@ -20,6 +20,45 @@ type service struct {
 
 }
 
+func (s service) EnableUser(id int)  error {
+	user, err := s.repository.FindByID(id)
+
+	if err != nil {
+		return err
+	}
+
+	if user.IsEnabled == false {
+		user.IsEnabled = true
+		_, err := s.repository.Update(*user)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	return errors.New("user is already enabled")
```

Comment:
@bukunmialuko .. if the user is already enabled don't return an error...return nil